src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Argument.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.plugins.report.jtreg.main.comparator;
+
+import java.util.Optional;
+
+public class Argument {
+    private final String argumentName;
+    private final String helpMessage;
+
+    private final String argumentFormat;
+
+    public Argument(String argumentName, String helpMessage, Optional<String> argumentFormat) {
+        this.argumentName = argumentName;
+        this.helpMessage = helpMessage;
+        this.argumentFormat = argumentFormat.orElse("");
```

Comment:
It took me a whille to find what the format is for and why it is optional. I'm not sure if jsut " <querystring>"  is worthy to be there. Maybe some example? NUmber, path ot Empty are usually ok...

Reply:
I redid the Argument class to not use Optional, I think it is simpler and makes more sense this way, what do you think? Also added some comments that should explain what the parameters are used for.